src/test/scala/karazin/scala/users/group/week3/homework/ProgramSuite.scala

```diff
@@ -13,8 +19,10 @@ import scala.concurrent.Future
 
 class ProgramSuite extends munit.FunSuite:
   
-  test("successful async test example") {
-    Future {
-      assertEquals(42, 42)
-    }
+  test("getPostView async test") {
+      val postView = getPostView(Post(UUID.randomUUID(),UUID.randomUUID()))
+      postView onComplete {
+        case Success(result) => assertEquals(42, 42)
```

Comment:
`assertEquals(42, 42)` always true, it checks nothing.
When you call `getPostView(Post(UUID.randomUUID(),UUID.randomUUID()))` you can memorize `postId` and `userId` and the compare with the results:
```
  case PostView(Post(postId, userId), _ :: Nil /*no-empty list of comments*/, _ :: Nil, _ :: Nil) => assertEquals(postId, ???); assertEquals(userId, ???); 
```